modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -81,6 +81,22 @@ you use the Preview version of `SidePanel` until the one in Main is replaced wit
 
 ---
 
+### Layout and Column
```

Comment:
Add this as a section under component deprecations in the table of content
![Screen Shot 2023-02-06 at 10 36 05 AM](https://user-images.githubusercontent.com/7966550/217044363-fd6a231b-95b6-4bd5-a00d-c86bae9a3963.png)